src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -1092,3 +1083,17 @@ function getStyles(theme: GrafanaTheme2) {
     }),
   };
 }
+
+export function wrapWildcardSearch(input: string) {
```

Comment:
Maybe these can be `service/query.ts` functions?

Reply:
Works for me. Done!